testhelpers/org.eclipse.n4js.tests.helper/src/org/eclipse/n4js/tests/helper/documentprovider/CountPostChangeBroadcastChangeNotificationManager.xtend

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2017 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.tests.helper.documentprovider
+
+import org.eclipse.core.internal.events.NotificationManager
+import org.eclipse.core.internal.events.ResourceChangeEvent
+import org.eclipse.core.internal.resources.Workspace
+import org.eclipse.core.internal.watson.ElementTree
+import org.eclipse.core.resources.IResourceChangeEvent
+
+/**
+  * This class counts the number of broadcastChange method calls.
+  */
+public class CountPostChangeBroadcastChangeNotificationManager extends NotificationManager {
```

Comment:
Why written in Xtend like java? Classes are always public by default, fields are private by default, methods are public by default. 0 is the default value for ints.

Reply:
Yeah, that's true. But occasionally I was told by other reviewers to explicitly state the accessibility modifier (public, private) for better readability. Nevertheless, I will adjust the code as you suggested.